scripts/release/make-release.py

```diff
@@ -104,9 +104,19 @@ def fetch():
         print("Writing release log...")
         release_message = make_release_message(new_tag, branches_and_tickets)
         write_release_log(release_message)
-        commit_tag_and_push()
+        commit_and_tag()
+
+        print("Updating YouTrack...")
+        tickets = list(ticket for branch, ticket in branches_and_tickets)
+        problems = tag_tickets(tickets, new_tag)
```

Comment:
I think this should not be done when `--test-run` is passed in (otherwise one can't do the "see what would be released").  If we do update the tags then it should ideally be done at the same time as a push so that the two don't get out of sync.  Otherwise if you make a release but don't go through and push it (but undo that commit to back out) the tickets are going to be in an inconsistent state.

Or I could have the logic wrong

Reply:
Agreed about `--test-run`, I'll fix that.

But yeah, this is tricky, because there's no way to make the change atomic across YouTrack. I figured that we didn't want to auto-push to GitHub, and we didn't want to have to manually run another step to update YouTrack.

This way, if you change your mind, you just need to find the tickets in YouTrack with the new tag, and revert them back to being empty, which is pretty simple. We could add another script to do that reversion for you, if it came up often.

Reply:
Adding test run seems enough for now - we can iterate over this until we work out something that works

Reply:
Fixed `--test-run`.